src/server/models/db/migrations/20230713125822_move_list_item_audits_to_events/migration.sql

```diff
@@ -0,0 +1,15 @@
+-- Move Reminder audit events to Event table: (https://www.db-fiddle.com/f/pAsG9CgsiMz5YMZZo9FJ1t/0)
+-- NOTE: Event table entries contain response data from Notify emails. Since Audit doesn't have this, it won't be included when entries are moved
+INSERT INTO "Event" ("time", "listItemId", "type", "jsonData")
+SELECT  a."createdAt" "createdAt",
+        (a."jsonData"->>'itemId')::int,
+        'REMINDER',
+        jsonb_set(
+          '{}'::jsonb,
+          '{reference}',
+          to_jsonb(a."jsonData"->'annualReviewRef')) ||
+          jsonb_build_object('notes', '{sendStartedProviderEmail}'::text[], 'eventName', '{reminder}'::text[])
```

Comment:
This can all be 
```
  jsonb_build_object(
     'reference', a."jsonData"->'annualReviewRef',
     'notes', '...',
     'eventName', '...' 
  )
```

then you can get rid of the jsonb_set